Src/IronPython/Compiler/Ast/ClassDefinition.cs

```diff
@@ -80,6 +80,9 @@ public class ClassDefinition : ScopeStatement {
         /// </summary>
         internal PythonVariable ModuleNameVariable { get; set; }
 
+        internal PythonVariable ClassCellVariable { get;set; }
+        internal PythonVariable ClassVariable { get;set; }
```

Comment:
Make these private unless there's a reason not to:
```suggestion
        private PythonVariable ClassCellVariable { get; set; }
        private PythonVariable ClassVariable { get; set; }
```